Docs/install/install_docker.rst

```diff
@@ -37,132 +37,255 @@
 .. _installation-docker:
 
 ##############################
-AIMET Installation in Docker
+AIMET Docker install and setup
```

Comment:
Again I feel "AIMET Docker install" may be a bit misleading. We are installing AIMET within a docker container, not installing an "AIMET docker".

Reply:
Changed.